tests/storage/snapshots/conftest.py

```
@@ -149,3 +167,195 @@ def vm_restore_with_predictable_names(
+
+    try:
+        with ThreadPoolExecutor(max_workers=NUM_MULTI_DISK_VMS) as executor:
+            futures = {
+                executor.submit(_restore_vm, (vm, snapshot)): vm for vm, snapshot in snapshots_of_four_vms.items()
+            }
+            for future in as_completed(futures):
+                vm = futures[future]
+                try:
+                    restores.append(future.result())
+                except Exception as error:
+                    LOGGER.error(f"Failed to restore VM {vm.name}: {error}")
```

Ruff (0.16.1)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)